integrations/jira/.port/resources/blueprints.json

```diff
@@ -16,6 +16,43 @@
     },
     "calculationProperties": {}
   },
+  {
+    "identifier": "jiraTeam",
+    "title": "Jira Team",
+    "icon": "Users",
+    "description": "A team within the organization",
+    "schema": {
+      "properties": {
+        "displayName": {
+          "title": "Display Name",
+          "type": "string",
+          "description": "The team's display name"
+        },
```

Comment:
let's remove this property since I see it's redundant according to the mapping. we're already mapping it to the entity title